.github/workflows/testool.yml

```diff
@@ -0,0 +1,53 @@
+name: Testool
+
+on:
+  schedule:
+    - cron: '30 23 * * *'
+  workflow_dispatch:
+    inputs:
+      suites:
+        description: "Suites"
+        required: true
+        default: "light"
```

Comment:
I see here its "light" as default, but in line 28  if `inputs.suites` is not set it uses "nightly".  Which is the suite that would be run every night from the scheduled job?

Reply:
When the workflow is being triggered by the scheduler, it uses the "nightly" suite:
```
      DATA: ${{ github.event.inputs.suites || 'nightly' }}
```

In case someone wants to trigger the workflow manually it gets by default "light" suite.

Reply:
Awesome, thanks!